jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/sso/bearer/CopyUserCodeForLoginDialog.kt

```diff
@@ -30,19 +33,29 @@ class CopyUserCodeForLoginDialog(
 
     private val pane = panel {
         row {
-            text(message("aws.sso.signing.device.code.copy.dialog.text"), maxLineLength = -1)
+            label(message("aws.sso.signing.device.code.copy.dialog.text"))
         }
 
         row {
             cell(
                 BorderLayoutPanel(5, 0).apply {
                     val action = CopyUserCodeForLogin(authCode)
-                    addToCenter(JBLabel(authCode).setCopyable(true))
+                    addToCenter(
+                        JBLabel(authCode).apply {
+                            tryOrNull {
+                                JBFont.create(JBFont.decode(EditorColorsUtil.getGlobalOrDefaultColorScheme().consoleFontName)).biggerOn(9f).asBold()
+                            }?.let {
+                                font = it
+                            }
+                            setCopyable(true)
+                        }
+                    )
                     addToRight(ActionButton(action, action.templatePresentation.clone(), ActionPlaces.UNKNOWN, ActionToolbar.NAVBAR_MINIMUM_BUTTON_SIZE))
```

Comment:
Should we remove this button now that we are only confirming and not copying?

Reply:
as discussed offline, keeping button but removing the default clipboard behavior